src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -65,8 +68,8 @@ def _complex_params(command_group, engine):
             c.argument('administrator_login', options_list=['--admin-user', '-u'])
             c.argument('administrator_login_password', options_list=['--admin-password', '-p'], required=False, help='The password of the administrator login.')
             c.argument('ssl_enforcement', arg_type=get_enum_type(['Enabled', 'Disabled']), options_list=['--ssl-enforcement'], help='Enable ssl enforcement or not when connect to server.')
-            c.argument('tier', arg_type=get_enum_type(['Basic', 'Standard']), options_list=['--performance-tier'], help='The performance tier of the server.')
-            c.argument('capacity', options_list=['--compute-units'], type=int, help='Number of compute units.')
+            c.argument('tier', arg_type=get_enum_type(['Basic', 'General Purpose', 'Standard']), options_list=['--performance-tier'], help='The performance tier of the server.')
```

Comment:
Test out "General Purpose"... argparse may not like that.  You might need to call it "GeneralPurpose" and convert it code if your service needs the string "General Purpose".

Reply:
Actually, it should be GeneralPurpose. Changed it